### PR TITLE
Refined repeat printing of a string

### PR DESCRIPTION
**Pull request by onebuck-code**
Example:
```
1) {RP:2:#}

_Output Result:_
##

2) Text{RP:2:*}Test string{RP:4:&*}

_Output Result:_
Text**Test string&*&*&*&*
```

### CODE REVIEW

**Comment by leesiongchan** [comment]
Thank you for your contribution, do you mind adding your new feature to the README as well? Thank again!

File: android/src/main/java/leesiongchan/reactnativeescpos/LayoutBuilder.java [hunk 1]
```diff
@@ -88,6 +92,29 @@ public String createAccent(String text, char accent, int charsOnLine) {
...
 
+    public String createCustomNumberOfString(String text) {
+        String symbol = "", count = "0";
+
+        if (text.contains("{RP:")) {
+            symbol = text.substring(0,text.indexOf("{RP:"));
+            count = text.substring(text.indexOf("{RP:") + 4, text.indexOf('}'));
+        }
+
+        if (text.contains("{C}")) {
```
Comment (leesiongchan) [inline android/src/main/java/leesiongchan/reactnativeescpos/LayoutBuilder.java]: Do we need this? I think {C} and {R} already been handled

Comment (onebuck-code) [reply android/src/main/java/leesiongchan/reactnativeescpos/LayoutBuilder.java]: Let me test and get back to you.
By logically it doesn't work

Comment (onebuck-code) [reply android/src/main/java/leesiongchan/reactnativeescpos/LayoutBuilder.java]: yes we need. u must include {C} {R}

File: android/src/main/java/leesiongchan/reactnativeescpos/LayoutBuilder.java [hunk 2]
```diff
@@ -142,6 +141,31 @@ public String createTextOnLine(String text, char space, String alignment, int ch
...
+    public String duplicateStringSymbol(String text) {
+        String repeatTag = "{RP:";
+        String regex = "\\"+repeatTag+"\\d+:.*?\\}";
+        Matcher m = Pattern.compile(regex).matcher(text);
+        int tagCount = 0;	
+		while(m.find()){
+			tagCount++;
+        }
+        
+        for(int x = 0; x < tagCount; x++){
```
Comment (leesiongchan) [inline android/src/main/java/leesiongchan/reactnativeescpos/LayoutBuilder.java]: Perhaps you can just use `split()` function the code should be much cleaner.

**Comment by leesiongchan** [comment]
Can you update your PR description also? It is outdated.

**Pull request merged**